solr/solrj-zookeeper/build.gradle

```diff
@@ -37,6 +37,13 @@ dependencies {
     implementation 'org.apache.httpcomponents:httpclient'
     implementation 'org.apache.httpcomponents:httpcore'
 
+
+    implementation('org.apache.curator:curator-client', {
+        exclude group: 'org.apache.zookeeper', module: 'zookeeper'
+    })
+    api('org.apache.curator:curator-framework', {
```

Comment:
implementation?

Reply:
As I mentioned before, it's in the API contract. Solr-core actually uses it.

Reply:
This is mostly a matter of taste; nothing is breaking based on this decision (I think).  Solr-core already has an explicit dependency so that point is moot.  I believe the impact would be some SolrJ user who adds solrj-zookeeper because they want to use ZK coordinates for CloudSolrClient.  Their runtime class path will have Curator either way.  But do we want such a user to have their compile-time class-path with Curator as well?  Like; are they *likely* to use Curator explicitly?  If we don't think it's *likely*, I think "implementation" best communicates an internal scope.

Reply:
Compilation vs runtime dependency doesn't have a huge effect on users, but I'll change it.

Reply:
Ok, tried it, and it won't work. Our `ZkACLProvider` is a sub-class of Curator's `ACLProvider`. Therefore, people using solrj-zookeeper and ACLs will be unable to compile without the api dependency. Changing back.